src/index.ts

```diff
@@ -249,6 +249,17 @@ export default class Collection<Resource extends { [key: string]: any }> {
   public find(id: string): Remote<Resource> {
     return safeGet(this.entities, id).getOrElse(RD.initial);
   }
+  public viewAsArray(ids?: string[]): Resource[] {
+    return this.view(ids).getOrElse([]);
+  }
+
+  public viewAtAsArray(at: string): Resource[] {
+    return this.viewAt(at).getOrElse([]);
+  }
+
+  public findAsNullable(id: string): Resource | null {
+    return this.find(id).toNullable();
```

Comment:
@alekhinen / @dylanpyle 

Now that `find` returns a `RemoteData`, do you think this method is all that useful?

```ts
const resource = entityStore.find(fooId).toNullable();
// vs
const resource = entityStore.findAsNullable(fooId);
```

Reply:
I don't know if I would ever remember to use `findAsNullable`. I'd prefer to just use `.find` where I need to.